src/helpers.ts

```diff
@@ -1078,6 +1080,11 @@ Plus they are only used when the contract is meant to be used as standalone when
     name: string,
     options: DeployOptions
   ): Promise<DeployResult> {
+    //log('fakeNewlyDeployed:', fakeNewlyDeployed, 'fakeNotNewlyDeployed:', fakeNotNewlyDeployed);
+    if (fakeNewlyDeployed || fakeNotNewlyDeployed) {
+      const deployment = await env.deployments.get(name);
+      return { newlyDeployed: fakeNewlyDeployed, ...deployment };
```

Comment:
I don't think this is a good approach.
newlyDeployed should always means the contract was newly deplioyed